DeusEx/Classes/DXRFixup.uc

```diff
@@ -155,15 +155,7 @@ function FixSamCarter()
 {
```

Comment:
maybe I should just remove fears for all important/invincible characters? I should probably remove fears from Paul too